pkg/controllers/management/secretmigrator/clusters_test.go

```diff
@@ -0,0 +1,175 @@
+package secretmigrator
```

Comment:
I would add some error-handling test cases. And maybe a couple more for some of the other secrets.